doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
+<p align="center"><b>Table 6-3-14-1:</b> VIM context Information.</p>
 
-<a name="6.3.13"></a>
-### 6.3.13 Proxy information
-Proxy information, this section could be empty if not needed.
+#### 6.3.14.2 Storage Context
+Parameters from storage vendor for integration.
+
+Only one entry is expected.
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| VENDOR | String | Yes |  |
+| AUTHORIZATION | String | Yes | One-way or Two-way authentication |
+| JOINT_WAY | String | Yes | by ISCSI or client |
+| DRIVER_FULL_NAME | String | Yes | Full path for storage driver |
+| CEPH_CONFIG_PATH | String | Yes | Full path for ceph.conf storage |
+| IS_PIM_JOINT | String | Yes | whether integrate with PIM, usaually "YES" for this value |
+| STORAGE_SOFTWARE_VERSION | String | Yes |  |
+
+<p align="center"><b>Table 6-3-14-2:</b> Storage context Information.</p>
 
+#### 6.3.14.3 Storage Client context
+This table defines the parameters for integration with storage client
+
+Multiple entries are expected, one entry for each authorization user.
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| JOINT_WAY | String | Yes | integration method for storage client, for example, RBD  |
+| COMPONENT_TYPE | String | Yes | for example: cinder, glance or nova |
+| AUTHORIZATION_USER | String | Yes | match with the component type |
+| KEYRING_FILENAME | String | Yes | Full path for keyring file, this should match the authentication user, |
+
+<p align="center"><b>Table 6-3-14-3:</b> Storage client context.</p>
+
+<a name="6.3.15"></a>
+### 6.3.15 Device Management information
+
+#### 6.3.15.1 SERVER PIM ACCOUNT
+Servers are managed by redfish, credentials should be the same for same type of device  
+
+Multiple entries are expected, one entry for each server model.
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| proxyAddress | String | Yes | Proxy address |
-| port | String | Yes | port |
-| user | String | Yes | user |
-| password | String | Yes | password |
+| VENDOR | String | Yes |  |
+| SERVER_MODEL | String | Yes | MODEL for each type of server |
+| REDFISH_USER | String | Yes |  |
+| REDFISH_PASSWD | String | Yes |  |
 
-<p align="center"><b>Table 6-3-13:</b> Proxy information.</p>
+<p align="center"><b>Table 6-3-15-1:</b> SERVER PIM ACCOUNT Information.</p>
+
+#### 6.3.15.2 Switch PIM Account
```

Comment:
"PIM" = ?

Reply:
Physical Infrastructure Manager
I have add the explanation

Reply:
Same comment as above

Reply:
The clear-text password issue?